examples/typescript/your_fungible_asset.ts

```diff
@@ -80,6 +81,34 @@ async function burnCoin(admin: Account, fromAddress: AccountAddress, amount: Any
   return pendingTxn.hash;
 }
 
+/**
+ * Fetches the name of the fungible asset created by the given account.
+ * This function interacts with the `get_name` view function in the Move module.
+ *
+ * @param aptos - An instance of the Aptos client.
+ * @param admin - The account that deployed the fungible asset module.
+ * @returns The name of the token as a string.
+ */
+async function fetchTokenName(aptos: Aptos, admin: Account): Promise<string> {
```

Comment:
These examples use a "global" aptos object, so it doesn't have to be passed here